src/App.tsx

```diff
@@ -76,109 +79,96 @@ end`
-            color="primary"
-            onClick={() => {
-              init().then(() => {
-                try {
-                  let resp = program(code, inputs, numOfOutputs);
-                  console.log(resp);
-                  setOutput({
-                    init: true,
-                    success: true,
-                    text: resp.join(" "),
-                  });
-                } catch (error) {
-                  setOutput({
-                    init: true,
-                    success: false,
-                    text: "Error: Check the developer console for details.",
-                  });
-                }
-              });
-            }}
-            endIcon={<SendIcon />}
-          >
-            Execute
-          </Button>
-          {output.init ? (
-            <Alert
-              variant="outlined"
-              severity={output.success ? "success" : "error"}
-              sx={{mx: 2}}
-            >
-              <Typography color="inherit" component="div">
-                {output.text}
-              </Typography>
-            </Alert>
-          ) : (
-            <></>
-          )}
-        </Box>
-        <Box sx={{ my: 4 }}>
-        <Typography color="primary" variant="h6">Inputs</Typography>
-          <CodeMirror
-            value={inputs}
-            height="100%"
-            maxHeight="200px"
-            theme={oneDark}
-            onChange={setInputs}
-          />
-        </Box>
-        <Box sx={{ my: 4 }}>
-        <Typography color="primary" variant="h6">Miden Assembly Code</Typography>
-          <CodeMirror
-            value={code}
-            height="100%"
-            theme={oneDark}
-            onChange={setCode}
-            maxHeight="500px"
-          />
-        </Box>
-
-      </Container>
+        <div className="flex items-center justify-between flex-wrap bg-blue-700 p-6">
+          <div className="flex items-center flex-shrink-0 text-white mr-6">
+              <MidenLogo />
+              <span className="font-semibold text-xl tracking-tight">Playground for Miden Examples in Miden Assembly
+              </span>
+          </div>
+          <div className="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
+            <div className="text-sm lg:flex-grow">
+              <a href="https://wiki.polygon.technology/docs/miden/user_docs/assembly/main" className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4">
+                Docs
+              </a>
+              <a href="https://github.com/0xPolygonMiden/examples#available-examples" className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4" target="_blank" rel="noopener noreferrer">
+                README: Examples
+              </a>
+              <a href="https://polygon.technology/solutions/polygon-miden/" className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white">
+                Homepage
+              </a>
```

Comment:
I would probably create a `Link` component or at least pull the custom styling out into a `globals.css` file so it's not duplicated. https://tailwindcss.com/docs/adding-custom-styles#adding-component-classes